matroska/19.md

```diff
@@ -6,8 +6,15 @@ ASCII-only data in string.
 
 ## Definition
 
-The string element is limited to certain byte ranges of ASCII plus a trailing optional null byte.
+The string element is limited to certain byte ranges of ASCII plus trailing optional null bytes.
 
-## Note
+## Reproducibility
+
+```sh
+./ffmpeg -f lavfi -i "color=c=black:size=2x2" -vframes 1 -pix_fmt gray8 -write_crc32 0 -c:v ffv1 -level 3 -slices 1 /tmp/EBML-ASCII-ONLY-IN-STRING-tmp.mkv
```

Comment:
These mkv outputs are directed to /tmp but other outputs are sent to ./
Also here uses ./ffmpeg and others use ffmpeg. Should be consistent.